### PR TITLE
Creates a dashboard page at /admin/temp/hoc_signups showing HOC 2015 signups.

### PR DESCRIPTION
**Pull request by ashercodeorg**
The dashboard displays a graph and a table of HOC 2015 signups, by day.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
@caleybrock Also for review, please confirm I'm not duplicating any existing work and that my choices are sane.

In particular, we have a cron job (analyze_hoc_activity) that runs hourly, grabbing these exact counts. However it only stores the aggregate, not the breakdown. An alternate implementation would be to cache the hourlies from the cron job and use that data to power this page. This seems overkill, as I don't expect this admin report page to be hit more than a few times per day. Though not cheap, the DB query isn't super expensive.

File: dashboard/config/routes.rb [hunk 1]
```diff
@@ -179,6 +179,7 @@ def redirect_to_teacher_dashboard
 
   # one-off internal reports
   get '/admin/brook/csppd', to: 'reports#csp_pd_responses', as: 'csp_pd_responses'
+  get '/admin/roxanne/hoc_signups', to: 'admin_reports#hoc_signups', as: 'hoc_signups'
```
Comment (Hamms) [inline dashboard/config/routes.rb]: can we reverse the trend of prefixing these routes with the name of the requester? Event if they're intended to be temporary views, I'd prever to have something like `/admin/temp/` and have a comment in the controller explaining the nature and source of the request.

Comment (caleybrock) [inline dashboard/config/routes.rb]: +1 on this idea, I think Alice and Laura were interested in this as well so it's not just for Roxanne

Comment (ashercodeorg) [inline dashboard/config/routes.rb]: Done. And +1. :)

**Comment by ashercodeorg** [comment]
AFAICT, the test failure in TestMilestoneParser is flaky and unrelated.

**Comment by Hamms** [comment]
LGTM! And confirmed; I've seen that failure before. It would likely vanish if you were to take the time to rerun the tests, but I don't think that's necessary

**Pull request merged**